### PR TITLE
First stab at adding the google analytics stuff. code needs to be upd…

### PR DESCRIPTION
**Pull request by roblevintennis**
needs the code updated. I basically monkey see'd https://github.com/mavenlink/mavenlink/commit/5ce4d79b6c34324aa0acd3a3ee47d29b03230a41 besides converting link.css from sass

https://www.pivotaltracker.com/story/show/163068777

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "extends": "mavenlint-react",
   "env": {
+    "browser": true,
```
Comment (ahuth) [inline .eslintrc]: I think we don't want to do this. Almost left a note about this, but we need our components to run on the server or the client, and `window` stuff in our components is a big no-no for that.

What we can do is have the `styleguide` directory have it's own eslint config that allows browser stuff

File: styleguide/content/utils/page-view.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import ReactGA from 'react-ga';
+
+ReactGA.initialize('UA-7529271-29');
```
Comment (ahuth) [inline styleguide/content/utils/page-view.js]: Per our discussion, I need to generate a new GA site/number.

Also, could perhaps look at setting it based off an ENV.

File: styleguide/components/styleguide/index.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
-import Styleguide from './styleguide';
+import Styleguide from './styleguide-container';
 
 export default Styleguide;
+
```
Comment (ahuth) [inline styleguide/components/styleguide/index.js]: Is this line intended?

Comment (roblevintennis) [reply styleguide/components/styleguide/index.js]: Not sure there

File: styleguide/components/link/link.css [hunk 4]
```diff
@@ -0,0 +1,16 @@
+@import "../../../src/styles/colors.css";
+
+.link {
+  &:link,
```
Comment (ahuth) [inline styleguide/components/link/link.css]: Convert to CSS:

```
.link:link, 
.link:visited {
  ...
}
```

File: styleguide/content/utils/page-view.js (hunk 2 above)
Comment (ahuth) [reply styleguide/content/utils/page-view.js]: @roblevintennis got a new number for ya:

```
UA-7529271-31
```

Comment (ahuth) [reply styleguide/content/utils/page-view.js]:
```suggestion
ReactGA.initialize('UA-7529271-31');
```

**Pull request merged**